aten/src/ATen/native/TensorIterator.h

```diff
@@ -316,6 +321,7 @@ struct CAFFE2_API TensorIterator {
   bool resize_outputs_ = true;
   bool is_reduction_ = false;
   bool compute_common_dtype_ = true;
+  bool compute_common_inputs_dtype_ = false;
```

Comment:
Please rename to `compute_common_dtype_only_for_inputs_` it will make code much more readable

Reply:
Renamed